src/components/custom-widget-elements/Checkbox.vue

```diff
@@ -72,7 +85,7 @@ watch(
 )
 
 const isInput = computed(() => {
-  return miniWidget.value.options?.dataLakeVariable?.persistent === true
+  return miniWidget.value.options.dataLakeVariable?.id || widgetStore.editingMode
```

Comment:
This seems wrong - the function is intended to check whether the connected variable is an input, that can accept user changes (i.e. not an output, like a MAVLink message field or BlueOS / Cockpit application status that is just being reported).

- The ID check will be true if there is any connected data lake variable (including output variables)
- The editingMode check seems to mean the widget should accept user control even in edit mode, which seems backwards, on top of (as I understand it) not belonging in this function?